src/EPR.RegulatorService.Facade.Core/Models/ReprocessorExporter/Registrations/MaterialsAuthorisedOnSiteDto.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace EPR.RegulatorService.Facade.Core.Models.ReprocessorExporter.Registrations;
+public class MaterialsAuthorisedOnSiteDto
```

Comment:
I know this is very minor, but should we adopt a convention of putting a line break between these two lines to be consistent with our formatting across the project (this applies to a couple of other files too)?
```suggestion
﻿namespace EPR.RegulatorService.Facade.Core.Models.ReprocessorExporter.Registrations;
﻿
public class MaterialsAuthorisedOnSiteDto
```